config/installers/rolebasedcreds/namespaced/kustomization.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Adds namespace to all resources.
+namespace: PLACEHOLDER-NAMESPACE
+namePrefix: sagemaker-k8s-operator-
+  
+bases:
+- ../../../base
+- resources
+
+resources:
+- ../../../rbac/namespaced
+
+patchesStrategicMerge:
+- resources/manager_add_service_account_patch.yaml
+
+patchesJson6902:
+# Rename created namespace
+- target:
+    version: v1
+    kind: Namespace
+    name: sagemaker-k8s-operator-system
+  patch: |-
+    - op: replace
+      path: "/metadata/name"
+      value: PLACEHOLDER-NAMESPACE
+# Patch broken ServiceAccount names/namespaces (they all default back to `default`)
```

Comment:
Can add comment explaining that the namespaces are set via line 5, whereas the names are reset to `default` and so must be set manually? By my reading I believe that is what is happening here. Perhaps even add a link to whatever Github issue you found so that we can remove this in future if Kustomize ever fixes it.

Nice fix.

Reply:
++ on comment for explanation.